prismic-configuration.js

```diff
@@ -1,24 +1,28 @@
-module.exports = {
+// node-fetch is used to make network requests to the Prismic Rest API.
+// In Node.js Prismic projects, you must provide a fetch method to the
+// Prismic client.
+import fetch from 'node-fetch';
+import * as prismic from '@prismicio/client';
 
-  apiEndpoint: 'https://your-repo-name.prismic.io/api/v2',
+export const repoName = 'my-repo-name'; // Fill in your repository name.
```

Comment:
Make sure to always use `your-repo-name`. This is what the `prismic theme` command looks for to replace.

Reply:
Changed from `my-repo-name` to `your-repo-name`.